lib/util/state.js

```diff
@@ -265,21 +269,18 @@ export const getActiveItineraries = createSelector(
   getActiveSearchNonRealtimeResponse,
   (state) => state.otp.filter,
   getActiveSearchRealtimeResponse,
-  (state) => state.otp.useRealtime,
   getActiveFieldTripRequest,
   (
     config,
     nonRealtimeResponse,
     itinerarySortSettings,
     realtimeResponse,
-    useRealtime,
     activeFieldTripRequest
   ) => {
-    // set response to use depending on useRealtime
     const response =
       !nonRealtimeResponse && !realtimeResponse
         ? null
-        : useRealtime
+        : realtimeResponse
         ? realtimeResponse
```

Comment:
Can this be simplified?

Reply:
Yep! in 23ef058a46835bd8ce5710176ca1671ffcf7a586